portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -52,10 +52,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       if (count >= numComments) break;
       
       long id = entity.getKey().getId();
+      String user = (String) entity.getProperty("user");
       String text = (String) entity.getProperty("text");
       long timestamp = (long) entity.getProperty("timestamp");
-
-      Comment comment = new Comment(id, text, timestamp);
+      long likes = (long) entity.getProperty("likes");
```

Comment:
`ClassCastException` is usually not readable and hard for debugging. Please do type check before case. Log warning or throw error if type does not match.